pkg/services/store/kind/dashboard/summary.go

```diff
@@ -29,6 +30,16 @@ func GetObjectSummaryBuilder() models.ObjectSummaryBuilder {
 // This implementation moves datasources referenced by internal ID or name to UID
 func NewStaticDashboardSummaryBuilder(lookup DatasourceLookup) models.ObjectSummaryBuilder {
 	return func(ctx context.Context, uid string, body []byte) (*models.ObjectSummary, []byte, error) {
+		var parsed map[string]interface{}
+		err := json.Unmarshal(body, &parsed)
+		if err != nil {
+			return nil, nil, err // did not parse
+		}
+		// values that should be managed by the container
+		delete(parsed, "uid")
```

Comment:
i don't have a good picture what other checks are in place, or if this is even possible, but should this be checked to see if it matches the expected `uid` and `version`?

Reply:
This is a case where existing dashboard JSON stores uid+version directly in the json blob, but we want to move to it being managed one level up.  

This removes them from the store blobs since they will be replaced on 'read'